src/N3Parser.js

```diff
@@ -614,6 +635,17 @@ export default class N3Parser {
     case ',':
       next = this._readObject;
       break;
+    case '{|':
+      if (!this._supportsRDFStar)
+        return this._error('Unexpected RDF-star syntax', token);
+
+      this._saveContext('{|', this._graph, this._subject, this._predicate, this._object);
+
+      // Note - we always use the default graph for the quoted triple component
```

Comment:
```suggestion
      // The spec mandates the default graph for the quoted triple component
```
(I assume that is true?)

Reply:
No - that goes back to https://github.com/rdfjs/N3.js/pull/311#issuecomment-1330008257 and [this discussion](https://github.com/rdfjs/types/issues/34). To summarize it:
 - The rdf-star specification only has a notion of [quoted triples](https://w3c.github.io/rdf-star/cg-spec/editors_draft.html#quoted-triples), not quoted quads. Quoted triples are not asserted in any graph; they are merely a subject or object term within another triple/quad.
 - The RDF/JS spec *only* has a notion of quads; not triples.
 
 Thus we need a convention of what to do with the graph term when an RDF/JS *quad* is describing an RDF-star quoted triple. The convention in this PR is to always use the default graph on quoted triples.